gradle/changelog.gradle

```diff
@@ -18,13 +22,15 @@ spotlessChangelog {
 	branch        'release'
 	tagPrefix     "${kind}/"
 	commitMessage "Published ${kind}/{{version}}" // {{version}} will be replaced
+	tagMessage "${kind} v{{version}}\n\n{{changes}}"
```

Comment:
This `${kind} v{{version}}\n` looks redundant.

<img width="800" alt="image" src="https://github.com/user-attachments/assets/e21c6be0-6e9c-477b-8700-fa0ced99967f">

<img width="800" alt="image" src="https://github.com/user-attachments/assets/acbb0752-e7ad-47d1-9949-bbf4b37b89b9">